lib/contracts/compiler.js

```diff
@@ -35,6 +38,9 @@ class Compiler {
         if (!matchingFiles || !matchingFiles.length) {
           return callback();
         }
+
+        let compiler = compilers[0];
```

Comment:
Are the plugins always gonna be at position 0?

Reply:
According to the current implementation, yes.